src/Microsoft.Framework.Caching.Abstractions/project.json

```diff
@@ -1,6 +1,10 @@
 {
     "dependencies": {
-        "Microsoft.Framework.NotNullAttribute.Sources": { "type": "build", "version": "1.0.0-*" }
+      "Microsoft.Framework.NotNullAttribute.Sources": {
+        "type": "build",
+        "version": "1.0.0-*"
+      },
+      "Newtonsoft.Json": "6.0.6"
```

Comment:
Stuff like this should go in optional add-on packages. Compare to https://github.com/aspnet/HttpAbstractions/blob/dev/src/Microsoft.AspNet.Http.Extensions/SessionExtensions.cs. Maybe M.F.C.Distributed.Serializers or .Extensions?